### DISCUSSION TITLE
How do hashtags work?

### DISCUSSION THREAD

**Discussion by kaffeeringe** [mastodon/mastodon#20171] [Q&A]
When I click on a hashtag in my timeline - what do I see? I assume all posts with this hashtag, that my instance knows of. It doesn't search all of the fediverse.

**Comment by trwnh** [comment]
That's correct

**Reply by kaffeeringe** [reply]
Thank you!

**Reply by kaffeeringe** [reply]
Which posts does my instance know of? I assume it knows only all posts that went through the timelines of its' users. So if nobody on your instance follows me. Your instance doesn't know my posts.

**Reply by FlominatorTM** [reply]
I think, but I'm not sure and I'm struggling to find anything on it, that as soon as somebody from your instance follows somebody from the other instance, that one is known.

**Reply by FlominatorTM** [reply]
I'm asking around:
https://genealysis.social/@Flominator/109539085735595424

**Comment by FlominatorTM** [comment]
See also https://github.com/mastodon/mastodon/discussions/20579
